app/views/course/announcements/_announcement.html.slim

```diff
@@ -0,0 +1,20 @@
+div class=("announcement #{ 'future' if announcement.valid_from > DateTime.now }")
+  h3
+    = announcement.title
+
+  - if can? :manage, Course::Announcement
+    .pull-right
```

Comment:
Both are actually okay to me. If I must say, I'd prefer `div.pull-right` - it feels slightly easier to identify the indentation plus we make it easier for new developers to understand the code. 

Just my 2 cents.